module/data/activity/heal-data.mjs

```diff
@@ -45,7 +45,7 @@ export default class HealActivityData extends BaseActivityData {
   getDamageConfig(config={}) {
     if ( !this.healing.formula ) return foundry.utils.mergeObject({ rolls: [] }, config);
 
-    const rollConfig = foundry.utils.mergeObject({ critical: { allow: false }, scaling: 0 }, config);
+    const rollConfig = foundry.utils.deepClone(config);
```

Comment:
Well we shouldn't be pulling out the critical part of that just to fix the scaling bug.

```suggestion
    const rollConfig = foundry.utils.mergeObject({ critical: { allow: false } }, config);
```